fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/impl/tool/attribute/AttributeManager.java

```diff
@@ -23,10 +23,12 @@
 
 import com.google.common.collect.HashMultimap;
 import com.google.common.collect.Multimap;
+import org.jetbrains.annotations.ApiStatus;
 
 import net.minecraft.entity.attribute.EntityAttribute;
 import net.minecraft.entity.attribute.EntityAttributeModifier;
 
+@ApiStatus.Internal
```

Comment:
General question: Do we want this on all internal classes?

If so would some way to enforce it for the whole of fabric api be a good idea?

Reply:
This could be applied via package info I assume

Reply:
Yes, that would be better.

Reply:
Fixed.